pkg/libinit/proc_linux.go

```diff
@@ -5,21 +5,21 @@
 package libinit
 
 import (
+	"golang.org/x/sys/unix"
 	"log"
 	"os"
 	"os/exec"
-	"syscall"
```

Comment:
run gofmt, this "unix" line is in the wrong place.

Reply:
gofmt doesn't fix this, you need to add an empty line after the stdlib imports and then the unix import. Gofmt only sorts imports within each block of imports separated by empty lines